Content.Shared/Clothing/EntitySystems/ClothingGrantingSystem.cs

```diff
@@ -0,0 +1,86 @@
+using Content.Shared.Clothing.Components;
+using Content.Shared.Inventory.Events;
+using Robust.Shared.Serialization.Manager;
+using Content.Shared.Tag;
+
+namespace Content.Shared.Clothing;
+// from SimpleStation 14
```

Comment:
license? i think adding that codebase's license up here would make sense since this was directly copied over